config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do 
   devise_for :users
   
+  post :incoming, to: 'incoming#create'
```

Comment:
@ianroberts131 agreed, the incoming route looks correct, as does related output from rake routes: 
```
incoming POST   /incoming(.:format)                            incoming#create
```

It may be that these changes are not on the production/Heroku environment.

Heroku uses the `master` branch. 

If these changes are not merged into master, go ahead and merge them into master and enter `git push heroku master` in the Terminal to get the latest `master` branch deployed to the production environment and see if the incoming route is now found and logs to the server logs.